client/js/api-reference/messages.mdx

```diff
@@ -191,9 +191,7 @@ try {
         print("RTVI client message:", msg.type, msg.data)
         if msg.type == "set-language":
             language = msg.data.get("language", "en-US")
-            await task.queue_frames([context_aggregator.user().get_context_frame()])
-        # Kick off the conversation
```

Comment:
This seemed to me to be an extraneous line of code that wasn't meant to be there. @mattieruth, could you confirm?

Reply:
maybe leave in the comment, though?

Reply:
but we _don't_ want to kick off the conversation here, right?